L1Trigger/L1TCalorimeter/python/caloParams_2018_v1_4_cfi.py

```diff
@@ -76,16 +76,16 @@
 caloStage2Params.jetLsb                = cms.double(0.5)
 caloStage2Params.jetSeedThreshold      = cms.double(4.0)
 caloStage2Params.jetNeighbourThreshold = cms.double(0.)
-caloStage2Params.jetPUSType            = cms.string("ChunkySandwich")
-caloStage2Params.jetPUSUseChunkySandwich = cms.uint32(True)
+caloStage2Params.jetPUSType            = cms.string("PhiRing1")
+caloStage2Params.jetPUSUsePhiRing      = cms.uint32(True)
 caloStage2Params.jetBypassPUS          = cms.uint32(0)
 
 # Calibration options
 caloStage2Params.jetCalibrationType = cms.string("LUT")
 
 caloStage2Params.jetCompressPtLUTFile     = cms.FileInPath("L1Trigger/L1TCalorimeter/data/lut_pt_compress_2017v1.txt")
 caloStage2Params.jetCompressEtaLUTFile    = cms.FileInPath("L1Trigger/L1TCalorimeter/data/lut_eta_compress_2017v1.txt")
-caloStage2Params.jetCalibrationLUTFile    = cms.FileInPath("L1Trigger/L1TCalorimeter/data/lut_calib_2018v1_ECALZS_noHFJEC.txt")
+caloStage2Params.jetCalibrationLUTFile    = cms.FileInPath("L1Trigger/L1TCalorimeter/data/lut_calib_2018v1_ECALZS_noHFJEC_HI.txt")
```

Comment:
@rekovic is this version specifically for HI only?

Reply:
@fabiocos 
Short answer is yes, but this config file is not used centrally, nor in any of the workflows.  It is used for offline expert analysis.

Reply:
Yes, this is the offline config that corresponds to the online settings for HI running. It won't be used anywhere unless called specifically when running the caloL2 emulator